backend/src/Request/OrderLogs/OrderLogsCreateRequest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace App\Request\Rate;
+
+use App\Entity\OrderEntity;
+use App\Entity\CaptainEntity;
+use App\Entity\StoreOwnerProfileEntity;
+use App\Entity\StoreOwnerBranchEntity;
+
+class OrderLogsCreateRequest
+{
+   private OrderEntity $orderId;
+
+   private StoreOwnerProfileEntity $storeOwnerProfile;
+   
+   private null|CaptainEntity $captainProfile;
+
+   private string $orderState;
+
+   private null|StoreOwnerBranchEntity $storeOwnerBranch;
```

Comment:
Why store branch may have null type while it is for sure there will be a branch to request an order from?